tests/random_schema.cc

```diff
@@ -263,6 +263,28 @@ std::vector<data_value> generate_frozen_tuple_values(std::mt19937& engine, value
     return values;
 }
 
+data_model::mutation_description::collection generate_user_value(std::mt19937& engine, const user_type_impl& type,
+        value_generator& val_gen) {
+    using md = data_model::mutation_description;
+
+    auto key_dist = std::uniform_int_distribution<uint16_t>(0, type.size() - 1);
+
+    std::map<uint16_t, md::atomic_value> collection;
+    for (size_t i = 0; i < type.size(); ++i) {
+        auto key = key_dist(engine);
+        auto member_type = type.type(key);
```

Comment:
This can generate the same value multiple times. Can a single UDT mutations contain multiple mutations for the same cell?

Reply:
Observe that I'm first generating a std::map which is then converted into a mutation. If there's a collision of keys during the generating phase (the for loop you are commenting), the old value will stay (the new value won't be inserted). That is equivalent to what you suggested in the previous comment: first generate a number of elements, then generate them without key collisions. I'm just doing it in a simpler way IMO and with a different distribution as a side effect (it's harder to have more non-null fields).

Reply:
Ok, makes sense.